x-pack/test_serverless/functional/test_suites/common/management/data_views/index.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../../../ftr_provider_context';
+
+export default ({ getService, loadTestFile }: FtrProviderContext) => {
+  describe('Data View Management', function () {
+    const esArchiver = getService('esArchiver');
+
+    before(async () => {
+      await esArchiver.unload('test/functional/fixtures/es_archiver/logstash_functional');
+      await esArchiver.loadIfNeeded('test/functional/fixtures/es_archiver/makelogs');
+    });
+
+    after(async () => {
+      await esArchiver.unload('test/functional/fixtures/es_archiver/makelogs');
+    });
+
+    loadTestFile(require.resolve('./serverless'));
+    loadTestFile(require.resolve('./_data_view_create_delete'));
+    loadTestFile(require.resolve('./_runtime_fields'));
+    loadTestFile(require.resolve('./_runtime_fields_composite'));
+    loadTestFile(require.resolve('./_exclude_index_pattern'));
+    loadTestFile(require.resolve('./_edit_field'));
+  });
```

Comment:
Would be great to also include `test/functional/apps/management/data_views/_index_pattern_filter.ts` at some point. It mainly checks that Data View Management page tabs work.

Reply:
No better time than now 🙂 added here: https://github.com/elastic/kibana/pull/164656/commits/ca7f2c79e1bc0601fa3ab69554a8d21fc68d7f26.